docs/src/tutorials/getting_started.ipynb

```diff
@@ -176,111 +148,89 @@
    "source": [
     "## Part 2: Using Operators\n",
     "\n",
-    "Now, we'll actually transform our temporal data using some operations:\n",
-    "- Using a moving average with a large window, the long-term `trend` slope is captured (sinusoid and noise average should be close to 0).\n",
-    "- The trend can be then subtracted from the original signal to get only the sinusoidal component `detrend`."
+    "Now, let's actually transform our data with a couple operations.\n",
+    "\n",
+    "First, using a moving average with a large window, the long-term `trend` is captured (sine and noise average should be close to 0)."
```

Comment:
"Captured" reads funny to me. What about:

To extract the trend, the sigmoid and noise signals are first removed using a moving average over a large moving window.